File: java-lhdiff-project/datasets/Sargun/file3_old.java
// File3 helper demo
public class file3 {
    public static void main(String[] args) {
        System.out.println("File3 demo starting.");
        int[] numbers = {2, 4, 6, 8, 10};
        printArray(numbers);
        int total = sum(numbers);
        double avg = average(numbers);
        System.out.println("Total: " + total);
        System.out.println("Average: " + avg);
        int target = 6;
        int index = indexOf(numbers, target);
        System.out.println("Index of " + target + ": " + index);
    }
    private static void printArray(int[] values) {
        for (int i = 0; i < values.length; i++) {
            System.out.println("Value " + i + ": " + values[i]);
        }
    }
    private static int sum(int[] values) {
        int result = 0;
        for (int value : values) {
            result += value;
        }
        return result;
    }
    private static double average(int[] values) {
        if (values.length == 0) {
            return 0.0;
        }
        int result = sum(values);
        return result / (double) values.length;
    }
    private static int indexOf(int[] values, int target) {
        for (int i = 0; i < values.length; i++) {
            if (values[i] == target) {
                return i;
            }
        }
        return -1;
    }
}

File: python-lhdiff-project/datasets/Sargun/File3_Old.py
# File3 helper demo
def print_array(values):
    for i, value in enumerate(values):
        print(f"Value {i}: {value}")
def sum_values(values):
    result = 0
    for value in values:
        result += value
    return result
def average(values):
    if len(values) == 0:
        return 0.0
    result = sum_values(values)
    return result / float(len(values))
def index_of(values, target):
    for i, value in enumerate(values):
        if value == target:
            return i
    return -1
def main():
    print("File3 demo starting.")
    numbers = [2, 4, 6, 8, 10]
    print_array(numbers)
    total = sum_values(numbers)
    avg = average(numbers)
    print("Total:", total)
    print("Average:", avg)
    target = 6
    index = index_of(numbers, target)
    print("Index of", target, ":", index)
    print("File3 demo finished.")
if __name__ == "__main__":
    main()
